src/applications/caregivers/components/FormPages/FacilityConfirmation.jsx

```diff
@@ -3,11 +3,32 @@ import PropTypes from 'prop-types';
 import FormNavButtons from 'platform/forms-system/src/js/components/FormNavButtons';
 
 const FacilityConfirmation = props => {
-  const { data, goBack, goForward } = props;
+  const { data, goBack, goForward, goToPath } = props;
   const selectedFacility = data['view:plannedClinic'].veteranSelected;
   const selectedCaregiverSupportFacility =
     data['view:plannedClinic'].caregiverSupport;
 
+  const isReviewPage = () => {
+    const urlParams = new URLSearchParams(window.location.search);
+    return urlParams.get('review') === 'true';
```

Comment:
Using `review` query param to determine if the Veteran is coming from the review page to determine where the `back` and `continue` buttons route to.